examples/dt/uni04delta/values-wo-ceph-client.yaml

```diff
@@ -0,0 +1,165 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: edpm-nodeset-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+
+data:
+  ssh_keys:
+    authorized: _replaced_
+    private: _replaced_
+    public: _replaced_
+
+  ceph:
+    conf: CHANGEME_CEPH_CONF
+    keyring: CHANGEME_CEPH_KEYRING
+
+  nova:
+    ceph:
+      conf: CHANGEME_NOVA_CEPH_CONF
+
+    migration:
+      ssh_keys:
+        private: _replaced_
+        public: _replaced_
+
+  nodeset:
+    ansible:
+      ansibleUser: cloud-admin
+      ansiblePort: 22
+      ansibleVars:
+        timesync_ntp_servers:
+          - hostname: clock.redhat.com
+
+        edpm_iscsid_image: '{{ registry_url }}/openstack-iscsid:{{ image_tag }}'
```

Comment:
same images and all references of registry_url and image_tag can be dropped

Reply:
I didn't want to loose the original file for the reference. I'll suggest to use the `backup: true` here:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/cifmw_ceph_client/tasks/edpm_values_post_ceph.yml#L23 and here:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/cifmw_ceph_client/tasks/edpm_service_values_post_ceph.yml#L23
 and use the same file name.

Reply:
Fixed the *-wo-* files. Will also make the kustomization dry run passing in the gate.